content/45.user_friendliness.md

```diff
@@ -63,6 +63,17 @@ We note in particular that in previous versions of yt, `algae` was known as `bds
 
 ### Enhanced Error Messages
 
+Providing helpful error messages is almost an art form; projects like @doi:10.3847/1538-4357/ac7c74 have developed complex, helpful and thoughtfully-designed methods of providing as much usable information as possible when something "goes wrong."
+In recent development history, we have spent a considerable amount of effort attempting to discern between errors that are "user-facing" and those that are strictly "internal."
+"Internal" errors, for the most part, are often highly-unexpected; they may result from malformed data, or data that yt (and the developers) do not know to expect, or from general software defects.
+"User-facing" errors are those that can be anticipated, and they are the errors for which we endeavor to provide helpful and extensive responses.
+
+Among others, some of the most useful error messages that yt provides are those related to accessing fields and loading datasets.
+For instance, accessing the field `"x_velocity"` instead of `"velocity_x"` will produce an error, suggesting the *correct* field to access.
+When loading a dataset, yt will attempt to determine what the format of the data is; if it is unable to unambiguously identify the data format, it will provide an error message that shows the different options, and indicates how to disambiguate.
+
+While these may seem like simple, obvious changes to make, they can hide difficult technical challenges, and more importantly, have dramatically improved the user experience for people using yt.
```

Comment:
this is the first time I'm getting "feedback" of sorts on this, much appreciated !